posts/inside-rust/2019-10-24-Clippy-removes-plugin-interface.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Clippy is removing its plugin interface"
+author: Philipp Krones
+description: "Now that compiler plugins are deprecated, Clippy is removing its deprecated plugin interface"
+team: the Dev tools team (Clippy) <https://www.rust-lang.org/governance/teams/dev-tools#clippy>
+---
+
+Today, we're announcing that Clippy will completely remove its plugin interface.
+Using the plugin interface has been deprecated for about one and a half year now
+([rust-lang/rust-clippy#2712]). Since then, unsilenceable warning have been
+emitted.  Now that compiler plugins are officially deprecated
+([rust-lang/rust#64675]), Clippy will remove its support for the plugin
+interface completely ([rust-lang/rust-clippy#4714]).
```

Comment:
Should we mention in which stable version this is likely going to be removed?

Reply:
This would be 1.40, right?